### ISSUE TITLE
Preview Plugin - Unable to download a selection of Files

### ISSUE THREAD

**Issue by samuel-kerrien** [BlueBrain/nexus#3158]
TL;DR

1. Disable Download button if `archives/write` permissions is not available on the target project.
2. Add tooltip when button is disabled with the same message
3. Use the following message on the tooltip (to remove the cart reference - such that it can be reused for both the cart and the data download): "You don’t have the required permissions to create an archive for some of the selected resources. Please contact your project administrator to request to be granted the required archives/write permission."

**Issue 1 - unable to download one or several files using the 'Download Selected Files'**
Using the Preview plugin, when selecting one or several file and then clicking on the button 'Download Selected Files', an error occur and the download isn't started.

![Screenshot 2022-03-16 at 17.25.44.png](https://images.zenhubusercontent.com/60ad1ee0d2c5ea18af754dc0/674f8878-315b-4ede-9972-4d424af6fb4a)

**Expected behavior**
The selected files should be downloaded as an archive.

**Issue 2 - info message mentions the data cart**
One additional issue, the information message when the button is disabled looks like it was copied from the data cart, see capture below (taken on Sandbox).

![Screenshot 2022-03-17 at 18.18.18.png](https://images.zenhubusercontent.com/60ad1ee0d2c5ea18af754dc0/8129223d-22b6-4e6b-836d-944e3120e53e)
